codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -205,8 +205,13 @@ update these file paths accordingly.
       );
     }
     directoriesVisited.add(directory.absolute.path);
-    final List<FileSystemEntity> entities = await directory.list().toList();
+    final List<FileSystemEntity> entities = await directory.list(followLinks: false).toList();
     for (FileSystemEntity entity in entities) {
+      if (entity is io.Link) {
+        log.info("current file or direcotry ${entity.path} is a symlink to ${await (entity as io.Link).target()}, "
```

Comment:
A call to `link.targetSync()` should be fast, and it will avoid needing to have an asynchronous suspension:

```suggestion
        log.info("current file or directory ${entity.path} is a symlink to ${(entity as io.Link).targetSync()}, "
```

Reply:
TIL. Didn't know targetSync is fast. Thanks for pointing this out Christopher!